libs/logic-apps-shared/src/utils/src/lib/models/connection.ts

```diff
@@ -57,6 +58,13 @@ export interface ConnectionProperties {
 export type Connection = ArmResource<ConnectionProperties>;
 
 export interface TestLink {
+  body?: string;
+  method: string;
+  requestUri: string;
+}
+
+export interface TestRequest {
+  body?: string;
   method: string;
   requestUri: string;
```

Comment:
TestLink and TestRequest are exact similar structures right? Do we really need a copy or can we just re-use one?